app/services/voucher_adjustments_service.rb

```diff
@@ -23,33 +23,35 @@ def self.calculate(order)
       handle_tax_included_in_price(order, amount)
     else
       adjustment.amount = amount
+      adjustment.save
```

Comment:
I wouldn't expect `save` in a `calculate` method.

Reply:
What would you do ? rename the method ? we need to save the result of the calculation at some point

Reply:
> I would probably call it `update`.

I think that's worth doing ☝️